### PR TITLE
Cleaned up Coreutils and changed ref

### PR DESCRIPTION
**Pull request by jemten**

### CODE REVIEW

File: lib/MIP/Gnu/Coreutils.pm [hunk 1]
```diff
@@ -0,0 +1,1013 @@
...
+    ## gnu_split
+    # Stores commands depending on input parameters
+    my @commands = qw(gnu_split);
+
+    ## Options
+    if ($lines) {
+        push @commands, '--lines=' . $lines;
+    }
+
+    if ($numeric_suffixes) {
```
Comment (henrikstranneheim) [inline lib/MIP/Gnu/Coreutils.pm]: It looks like the $numeric_suffixes are evaluated twice in gnu_split

Comment (henrikstranneheim) [inline lib/MIP/Gnu/Coreutils.pm]: Ok, now I remember it is supposed to be like that, but could you remove one if($numeric_suffixes) and move the 
```
push @commands, '--suffix-length=' . $numeric_suffixes;
```
to L781.

File: mip.pl [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env perl
+!/usr/bin/env perl
```
Comment (henrikstranneheim) [inline mip.pl]: You need to add the '#' again

Comment (jemten) [inline mip.pl]: Strange. I thought I added that... It's part of my local file. Will update

File: lib/MIP/Gnu/Coreutils.pm [hunk 3]
```diff
@@ -0,0 +1,1013 @@
...
+            default     => 0,
+            allow       => [ 0, 1 ],
+            strict_type => 1,
+            store       => \$verbose
+        },
+    };
+
+    check( $tmpl, $arg_href, 1 ) or croak qw[Could not parse arguments!];
+
+    my $SPACE = q{ };
```
Comment (henrikstranneheim) [inline lib/MIP/Gnu/Coreutils.pm]: Let's start making the constants proper globals and just define them once per perl module

Add: 
```
## Constants
 my $SPACE = q{ };
```
etc after the begin block

**Pull request merged**